index.js

```diff
@@ -82,6 +82,14 @@ module.exports = (input, opts) => {
 			})
 		},
 		{
+			title: 'Pre-release version check',
+			task: () => {
+				if ((VERSIONS.slice(3).includes(input) || semver.prerelease(input)) && !opts.tag) {
```

Comment:
Would also be clearer to manually define the array `'premajor', 'preminor', 'prepatch', 'prerelease'` in a local variable here.